action.yml

```diff
@@ -39,14 +39,44 @@ inputs:
     default: ""
   vcs_type:
     description: Terraform VCS type (e.g., "github"). Superseded by `vcs_token_id`. If neither are passed, no VCS integration is added.
+    required: false
   vcs_token_id: 
     description: Terraform VCS client token ID. Takes precedence over `vcs_name`. If neither are passed, no VCS integration is added.
+    required: false
   vcs_repo:
     description: Repository identifier for a VCS integration. Required if `vcs_name` or `vcs_token_id` are passed
     default: "${{ github.repository }}"
   vcs_ingress_submodules:
     description: Whether to allow submodule ingress
     default: false
+  agent_pool_id: 
+    description: ID of an agent pool to assign to the workspace. If passed, execution_mode is set to "agent".
+    required: false
+  execution_mode:
+    description: Execution mode to use for the workspace
+    required: false
+  global_remote_state: 
+    description: Whether all workspaces in the organization can access the workspace via remote state
+    default: true
```

Comment:
I believe the default for this is now false for new TF Cloud orgs. There's an org default in TFE.